build/templates/backend/scripts/optimism_testnet_archive.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+{{define "main" -}}
+
+set -e
+
+GETH_BIN={{.Env.BackendInstallPath}}/{{.Coin.Alias}}/geth
+DATA_DIR={{.Env.BackendDataPath}}/{{.Coin.Alias}}/backend
+
+CHAINDATA_DIR=$DATA_DIR/geth/chaindata
+SNAPSHOT=https://storage.googleapis.com/oplabs-goerli-data/goerli-bedrock.tar
```

Comment:
there is not an archive snapshot available and I can't find a testnet genesis file, so this isn't the most accurate. we can just remove the archive testnet if you would like, let me know your thoughts